ui/components/fixture-capability-table.vue

```diff
@@ -157,6 +157,15 @@ export default {
         }
       );
     }
+  },
+  methods: {
+    getColorDescription: capability => {
```

Comment:
[Don't use arrow functions for Vue's methods / computed properties](https://vuejs.org/v2/api/#methods)